v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/migrations/avro/AvroToValueMapper.java

```diff
@@ -88,6 +88,9 @@ static Map<Type, AvroToValueFunction> getGsqlMap() {
     gsqlFunctions.put(
         Type.int64(),
         (recordValue, fieldSchema) -> Value.int64(avroFieldToLong(recordValue, fieldSchema)));
+    gsqlFunctions.put(
+        Type.float32(),
+        (recordValue, fieldSchema) -> Value.float64(avroFieldToFloat(recordValue, fieldSchema)));
```

Comment:
Should this be Value.float32(... on line 93?

Reply:
Thanks for catching this. Had missed it.
Fixed in current push.